### PR TITLE
update studio

### PR DESCRIPTION
**Pull request by foesa-yang**

### CODE REVIEW

File: docs-2.0/nebula-studio/deploy-connect/st-ug-deploy.md [hunk 1]
```diff
@@ -48,7 +48,8 @@ Nebula Graph Studio（ 以下简称 Studio ）支持云端或本地部署。云
    当屏幕返回以下信息时，表示 PRM 版 Studio 已经成功启动。
 
    ```bash
-   Created symlink from /etc/systemd/system/multi-user.target.wants/nebula-graph-studio.service to /usr/lib/systemd/system/nebula-graph-studio.service.
+   Nebula Studio has been installed.
```
Comment (hetao92) [inline docs-2.0/nebula-studio/deploy-connect/st-ug-deploy.md:51]: Start installing Nebula Studio now...

File: docs-2.0/nebula-studio/quick-start/st-ug-import-data.md [hunk 2]
```diff
@@ -39,6 +40,11 @@
 1. 在顶部导航栏里，点击 **导入** 页签。
 2. 在标签页内点击 **导入数据** 按钮。
 3. 在 **导入数据** 页面，点击 **导入模版** 直接通过配置文件导入数据或点击 **+ 创建导入任务** 按钮，完成以下任务：
+  
+  !!! caution
+
+        Studio 通过 **导入模板** 导入数据与 [Nebula Importer](../../nebula-importer/use-importer.md) 大致相同，但是 **导入模板** 中配置文件的所有文件路径仅保留文件名。并且请确保在导入 YAML 文件之前上传所有 CSV 数据文件。
```
Comment (abby-cyber) [inline docs-2.0/nebula-studio/quick-start/st-ug-import-data.md:46]:
```suggestion
        Studio 通过**导入模板**导入数据与 [Nebula Importer](../../nebula-importer/use-importer.md) 大致相同，但是**导入模板**中配置文件的所有文件路径仅保留文件名。并且请确保在导入 YAML 文件之前上传所有 CSV 数据文件。
```

File: docs-2.0/nebula-studio/about-studio/st-ug-what-is-graph-studio.md [hunk 3]
```diff
@@ -19,11 +19,11 @@ Nebula Studio（简称 Studio）是一款可以通过 Web 访问的开源图数
 
 Studio 可以方便管理管理 Nebula Graph 图数据，具备以下功能：
 
-  - 使用 **Schema** 管理功能，用户可以使用图形界面完成图空间、Tag（标签）、Edge Type（边类型）、索引的创建，查看图空间的统计数据，快速上手 Nebula Graph。
+- 使用 **Schema** 管理功能，用户可以使用图形界面完成图空间、Tag（标签）、Edge Type（边类型）、索引的创建，查看图空间的统计数据，快速上手 Nebula Graph。
   
-  - 使用 **导入** 功能，通过简单的配置，用户即能批量导入点和边数据，并能实时查看数据导入日志。
+- 使用 **导入** 功能，通过简单的配置，用户即能批量导入点和边数据，并能实时查看数据导入日志。
```
Comment (abby-cyber) [inline docs-2.0/nebula-studio/about-studio/st-ug-what-is-graph-studio.md:24]:
```suggestion
- 使用**导入**功能，通过简单的配置，用户即能批量导入点和边数据，并能实时查看数据导入日志。
```

File: docs-2.0/nebula-studio/about-studio/st-ug-what-is-graph-studio.md [hunk 4]
```diff
@@ -19,11 +19,11 @@ Nebula Studio（简称 Studio）是一款可以通过 Web 访问的开源图数
...
 
-  - 使用 **控制台** 功能，用户可以使用 nGQL 语句创建 Schema，并对数据执行增删改查操作。
+- 使用 **控制台** 功能，用户可以使用 nGQL 语句创建 Schema，并对数据执行增删改查操作。
```
Comment (abby-cyber) [inline docs-2.0/nebula-studio/about-studio/st-ug-what-is-graph-studio.md:26]:
```suggestion
- 使用**控制台**功能，用户可以使用 nGQL 语句创建 Schema，并对数据执行增删改查操作。
```

**Review by abby-cyber — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

**Pull request merged**